src/store/tribeModule.js

```diff
@@ -0,0 +1,18 @@
+export const playerTribe = {
+    state: () => ({
+        name: "NameOfTheTribe",
+        description: "DescOfTheTribe",
+        points: 50000
+    }),
+    getters: {
+        playerTribe(state) {
+            return {
+                name: state.name,
+                description: state.description,
+                points: state.points
+            }
+        }
+    },
+    mutations: {},
+    actions: {}
```

Comment:
what with mutations and actions? It's possible for you to do them, predicting the actions you want to execute on these entities